packages/ui/rollup.dts.config.ts

```diff
@@ -0,0 +1,35 @@
+import { defineConfig, type Plugin } from 'rollup'
+import dtsPlugin from 'rollup-plugin-dts'
+import aliasPlugin from '@rollup/plugin-alias'
+import path from 'path'
+
+const shimCssPlugin = (): Plugin => {
+  const CSS_SUFFIX = `.css.d.ts`
+
+  return {
+    name: 'soramitsu-ui:shim-css',
+    resolveId(id) {
+      if (/\.(css|scss|sass)$/.test(id)) return id + CSS_SUFFIX
+    },
+    load(id) {
+      if (id.endsWith(CSS_SUFFIX)) return `declare const css: string\n export default css`
+    },
+  }
+}
+
+export default defineConfig({
```

Comment:
Hmm, I wonder whether `auto-imports.d.ts` is considered by `rollup-plugin-dts` or not...

Reply:
Fixed with https://github.com/soramitsu/soramitsu-js-ui-library/pull/502/commits/af6ac344156d950859936944e387eeebb6ff665e